src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -22,9 +22,9 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
```

Comment:
🤔  I think there is some glitch in Rider: When I add new files I also always have this change, where an empty string is added to before the `/>`. But once you auto reformat the file it's gone, so according to the styling guidelines it shouldn't be added. Therefore I always revert those changes, I'd also suggest to do the same here. Maybe we should figure out why this happens.

Reply:
I reverted Elastic.Apm.csproj to its original state (without space before `/>`) but when I tried to format in Rider it kept the  spaces before `/>`

Reply:
That's strange. When I format it in Rider it always removed the space before `/>`. Also tried to specify this in the `.editorconfig`, but it does not seem to work. https://developercommunity.visualstudio.com/content/problem/159834/editorconfig-doesnt-work-for-csproj-files.html 
Is maybe my setup wrong and the default is that there should be a space before the `/>`?

ElasticSearch-net also has those spaces: https://github.com/elastic/elasticsearch-net/blob/master/src/Elasticsearch.Net/Elasticsearch.Net.csproj

Reply:
Update: I think this was my fault... 👿   I found a setting that controls this:
<img width="1263" alt="Screenshot 2019-03-25 at 11 50 01" src="https://user-images.githubusercontent.com/1091853/54914382-1b17a800-4ef5-11e9-8676-954cf85708fe.png">

I suspect the default is `true`, but somehow in my case it was `false`. 

I'll try to figure out how to set this in `.editorconfig` and I'll try to update the whole repo accordingly. If I fail, then I'll simply also set it to `true` for me, update the repo, and we'll continue that way. For this PR I'd say we can go on with whatever option... space or not, doesn't matter, we'll probably udpate this long term.